frontend/packages/dsco/monoweave.config.mjs

```diff
@@ -0,0 +1,18 @@
+/**
+ * Monoweave Configuration File for automated CHANGELOG and version bumping
+ *
+ * See: https://monoweave.github.io/monoweave/configuration
+ */
+
+/** @type {import('@monoweave/types').MonoweaveConfigFile} */
+export default {
+  preset: 'monoweave/preset-recommended',
+
+  // Aggregates changelog entries to a single file instead of the "recommended"
```

Comment:
Is `monoweave` typically installed in the monorepo root (rather than per package)?

Reply:
I believe we're only looking to have CHANGELOG for DSCO. If we roll it out to other packages then we can consider installing at the root.